modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultGenerator.java

```diff
@@ -17,6 +17,8 @@
 
 package org.openapitools.codegen;
 
+import com.google.common.collect.Lists;
```

Comment:
Is `google.common.colllect` preferred over `java.util.*` in OAI generator?

Reply:
Just an old habit from other projects. I used the basic java methods to create new Lists and Maps.